nbstripout/_utils.py

```diff
@@ -140,15 +177,26 @@ def strip_output(
         # Remove the outputs, unless directed otherwise
         if 'outputs' in cell:
             # Default behavior (max_size == 0) strips all outputs.
-            if not keep_output_this_cell:
-                cell['outputs'] = [output for output in cell['outputs'] if get_size(output) <= max_size]
+            if not keep_output_this_cell or keep_output_types:
+                cell['outputs'] = [output for output in cell['outputs']
+                                   if get_size(output) <= max_size
```

Comment:
Should this actually take precedence? I don't mind either way but suggest documenting in the README and/or docstring.